### PR TITLE
K8SPSMDB-389 document pitr

### PR DESCRIPTION
**Pull request by fiowro**
[K8SPSMDB-389](https://jira.percona.com/browse/K8SPSMDB-389) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

### CODE REVIEW

File: source/backups.rst [hunk 1]
```diff
@@ -135,9 +135,47 @@ The example of such file is `deploy/backup/restore.yaml <https://github.com/perc
...
+message in the operator logs.
+
+.. note:: Adding a new bucket when point-in-time recovery is enabled will not
+   break it, but put error message about the additional bucket in the operator
+   logs as well.
+
 Restore the cluster from a previously saved backup
 --------------------------------------------------
 
+The Operator supports the ability to perform a full restore on a PostgreSQL
```
Comment (spron-in) [inline source/backups.rst:171]:
```suggestion
The Operator supports the ability to perform a full restore on a MongoDB
```

File: source/backups.rst [hunk 2]
```diff
@@ -176,6 +214,60 @@ Following steps are needed to restore a previously saved backup:
               backupName: backup1
             EOF
 
+.. _backups-pitr-restore:
+
+Restoring backup with point-in-time recovery
+********************************************
+
+Following steps are needed to to roll back the cluster to a
```
Comment (spron-in) [inline source/backups.rst:222]:
```suggestion
Following steps are needed to roll back the cluster to a
```

File: source/backups.rst [hunk 3]
```diff
@@ -176,6 +214,60 @@ Following steps are needed to restore a previously saved backup:
...
+specific date and time:
+
+Details:
+
```
Comment (spron-in) [inline source/backups.rst:226]:
```suggestion
```

Comment (spron-in) [reply source/backups.rst:226]: I don't understand why we need Details here :)

File: source/backups.rst [hunk 4]
```diff
@@ -176,6 +214,60 @@ Following steps are needed to restore a previously saved backup:
...
+1. First of all make sure that the cluster is running.
+
+2. Now find out correct name for the **cluster** (backup name is not needed if
+   point-in-time recovery is enabled and will be ignored). Available
+   clusters can be listed with the following command:
```
Comment (spron-in) [inline source/backups.rst:231]: This is not right. In the spec we say that backupName is needed. We do that to ensure that user knows which backup to use and where oplogs are. It is to cover the case where user adds new S3 bucket if PITR is enabled and some other weird  cases.

File: source/backups.rst [hunk 5]
```diff
@@ -176,6 +214,60 @@ Following steps are needed to restore a previously saved backup:
...
+            kind: PerconaServerMongoDBRestore
+            metadata:
+              name: restore1
+            spec:
+              clusterName: my-cluster-name
+              pitr:
+                type: date
+                date: YYYY-MM-DD hh:mm:ss
+            EOF
+
```
Comment (spron-in) [inline source/backups.rst:270]: I would love to mention here, that the user needs to take full backup for PITR to function. PBM will not opload if there is no full backup.
This is true for new clusters and also true for clusters which just recovered from backup.

**Review by spron-in — CHANGES_REQUESTED** [review]

**Review by dAdAbird — APPROVED** [review]

**Review by tplavcic — APPROVED** [review]

**Review by spron-in — APPROVED** [review]

**Pull request merged**